gui/drawer.go

```diff
@@ -14,20 +15,31 @@ import (
 
 type drawer struct {
 	widgets   *widgets
+	gridOpts  *gridOpts
 	chartCh   chan *attacker.Result
 	gaugeCh   chan bool
 	metricsCh chan *attacker.Metrics
 
 	// aims to avoid to perform multiple `redrawChart`.
-	chartDrawing bool
+	chartDrawing int32
```

Comment:
nice catch! can you use `go.uber.org/atomic.Bool` instead of sync/atomic?

https://godoc.org/go.uber.org/atomic#Bool

Reply:
sure :) I thought of using it but as I wasn't sure if you'd be ok with one more dependency I just left it at stdlib. I'll update shortly

Reply:
Thanks for your concern :)

Reply:
Done. Using Uber's atomic.

Also, when running with `-race`  and doing a Ctrl-C, after a attack I believe there is a data race in termdash:
```go
WARNING: DATA RACE
Write at 0x000000c5e448 by main goroutine:
  github.com/nsf/termbox-go.Close()
      /home/brenol/goworkspace/pkg/mod/github.com/nsf/termbox-go@v0.0.0-20200204031403-4d2b513ad8be/api.go:149 +0x5e8
  github.com/mum4k/termdash/terminal/termbox.(*Terminal).Close()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/terminal/termbox/termbox.go:163 +0x55
  github.com/nakabonne/ali/gui.Run()
      /home/brenol/goworkspace/src/github.com/brenol/ali/gui/gui.go:34 +0x230
  main.(*cli).run()
      /home/brenol/goworkspace/src/github.com/brenol/ali/main.go:117 +0x26c
  main.main()
      /home/brenol/goworkspace/src/github.com/brenol/ali/main.go:60 +0x104

Previous read at 0x000000c5e448 by goroutine 18:
  github.com/nsf/termbox-go.flush()
      /home/brenol/goworkspace/pkg/mod/github.com/nsf/termbox-go@v0.0.0-20200204031403-4d2b513ad8be/termbox.go:242 +0x3e
  github.com/nsf/termbox-go.Flush()
      /home/brenol/goworkspace/pkg/mod/github.com/nsf/termbox-go@v0.0.0-20200204031403-4d2b513ad8be/api.go:211 +0x784
  github.com/mum4k/termdash/terminal/termbox.(*Terminal).Flush()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/terminal/termbox/termbox.go:113 +0x2f
  github.com/mum4k/termdash.(*termdash).redraw()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/termdash.go:278 +0x1bd
  github.com/mum4k/termdash.(*termdash).evRedraw()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/termdash.go:294 +0xa6
  github.com/mum4k/termdash.(*termdash).subscribers.func3()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/termdash.go:230 +0x44
  github.com/mum4k/termdash/private/event.(*subscriber).callback()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/private/event/event.go:95 +0x5a
  github.com/mum4k/termdash/private/event.(*subscriber).run()
      /home/brenol/goworkspace/pkg/mod/github.com/mum4k/termdash@v0.12.2/private/event/event.go:110 +0xd2
```

Reply:
Thank you for letting me know. Yup, I could repro and this is obviously a different problem. Let's address it as another issue 👍

Reply:
Open an issue about it: https://github.com/nakabonne/ali/issues/69